.github/workflows/build.yml

```diff
@@ -298,3 +298,73 @@ jobs:
         if-no-files-found: error
         retention-days: 30
 
+  archlinux:
+    strategy:
+      matrix:
+        arch:
+          [
+            "linux/amd64 x86_64",
+            "linux/arm64/v8 aarch64",
+          ]
+    name: "Archlinux ${{ matrix.arch }}"
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+      with:
+        platforms: all
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v1
+      with:
+        version: latest
+    - name: Read info
+      id: tags
+      shell: bash
+      run: |
+        arch="${{ matrix.arch }}"
+        echo ::set-output name=PLATFORM::${arch%% *}
+        echo ::set-output name=ARCH::${arch##* }
```

Comment:
Could these two env vars be maybe also inferred via `uname` too? If so they could be exported via the `set-vars.sh` script, too.
`uname -m` yields `x86_64` for me and `uname -o` yields `GNU/Linux`

Reply:
It could be posible I have just taken the ci code from [paru](https://github.com/Morganamilo/paru/blob/master/.github/workflows/build.yml) and I would say not as we do not run on that architect at that point but are using docker cross architect builds for it. So in there we could get those values from.

Reply:
I think there's no need to torture you. I can't await playing around with Arch Linux soon myself, too. :)